integration_tests/suite/test_cdr.py

```diff
@@ -730,15 +827,21 @@ def test_given_token_with_no_user_uuid_when_list_my_cdr_then_400(self):
 
     @call_logs([
         {'date': '2017-04-10'},
-        {'date': '2017-04-11', 'participants': [{'user_uuid': USER_1_UUID}]},
-        {'date': '2017-04-12', 'participants': [{'user_uuid': USER_1_UUID}]},
-        {'date': '2017-04-13', 'participants': [{'user_uuid': USER_1_UUID}]},
-        {'date': '2017-04-14', 'participants': [{'user_uuid': USER_1_UUID}]},
-        {'date': '2017-04-15', 'participants': [{'user_uuid': USER_2_UUID}]},
+        {'date': '2017-04-11', 'participants': [{'user_uuid': USER_1_UUID,
+                                                 'tenant_uuid': VALID_TENANT}]},
+        {'date': '2017-04-12', 'participants': [{'user_uuid': USER_1_UUID,
+                                                 'tenant_uuid': VALID_TENANT}]},
+        {'date': '2017-04-13', 'participants': [{'user_uuid': USER_1_UUID,
+                                                 'tenant_uuid': VALID_TENANT}]},
+        {'date': '2017-04-14', 'participants': [{'user_uuid': USER_1_UUID,
+                                                 'tenant_uuid': VALID_TENANT}]},
+        {'date': '2017-04-15', 'participants': [{'user_uuid': USER_2_UUID,
+                                                 'tenant_uuid': VALID_TENANT}]},
```

Comment:
If you do not care about the tenant_uuid in your test, then you can add the default tenant_uuid: VALID_TENANT in the fixture of call_log when a user_uuid is set. It will be less verbose and more readable

Reply:
I will do that.